doc/radosgw/s3-notification-compatibility.rst

```diff
@@ -105,6 +105,12 @@ Event Types
 | ``s3:ReducedRedundancyLostObject``           | Not applicable to Ceph                                      |
 +----------------------------------------------+-----------------+-------------------------------------------+
 
+Note::
+------
```

Comment:
did you try to render the doc?
i think you should remove this line, and indent the actual note